regression/regression_utils.py

```diff
@@ -31,6 +32,38 @@
 """
 
 
+class RetryTestsMetaclass(type):
+
+    NUM_RETRIES = 2
+    FLAKY_ERROR_CLASSES = (AssertionError,)
```

Comment:
Hmm, I have a suspicion that the retry count and flaky class set should be configurable on derived classes.

I'm also not sure why `AssertionError` is flaky -- I would expect stuff related to HTTP error response codes, but not assertions.